pyspark/test/bigdl/test_simple_integration.py

```diff
@@ -533,7 +533,7 @@ def test_save_jtensor_dict(self):
         tensors["tensor1"] = JTensor.from_ndarray(np.random.rand(3, 2))
         tensors["tensor2"] = JTensor.from_ndarray(np.random.rand(3, 2))
         # in old impl, this will throw an exception
-        _py2java(self.sc, tensors)
+        _py2java(self.sc._gateway, tensors)
```

Comment:
add unittest for the new added ModelBroadcast?

Reply:
added